app/views/admin/forms/permissions.html.erb

```diff
@@ -27,8 +27,8 @@
 
     $("#add-user-button").on("click", function(event) {
       event.preventDefault();
-      var userId = $("#add-user-id").val();
```

Comment:
use snake case naming to match conventional Rails label tags. ultimately, improves accessibility of labels to inputs.